docs/source/format/Columnar.rst

```diff
@@ -1053,7 +1053,7 @@ the dictionaries can be properly interpreted: ::
 
 The dictionary ``id`` in the message metadata can be referenced one or more times
 in the schema, so that dictionaries can even be used for multiple fields. See
-the :doc:`Layout` document for more about the semantics of
+the `Dictionary-encoded Layout`_ section for more about the semantics of
```

Comment:
Could you add a label to that section and use `` :ref`..` `` syntax instead? That's more robust in case the title is changed (see https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations, and its recommendation to use that instead of standard links)

Reply:
Makes sense, I have revised accordingly